recipe_scrapers/emmikochteinfach.py

```diff
@@ -0,0 +1,40 @@
+from ._abstract import AbstractScraper
```

Comment:
```suggestion
# mypy: allow-untyped-defs

from ._abstract import AbstractScraper
```

Adding this to the top of the file should resolve the `mypy` check warnings.  Looks like I forgot to update the scraper template to add this..